widget/accordioncontainer_internal_test.go

```diff
@@ -0,0 +1,292 @@
+package widget
+
+import (
+	"testing"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/test"
+	"fyne.io/fyne/theme"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccordionContainer_toggleForIndex(t *testing.T) {
+	ai := NewAccordionItem("foo", NewLabel("foobar"))
+	ac := NewAccordionContainer(ai)
+	assert.False(t, ai.Open)
+	toggle := ac.toggleForIndex(0)
+	toggle()
+	assert.True(t, ai.Open)
+	toggle()
+	assert.False(t, ai.Open)
+}
+
+func TestAccordionContainerRenderer_MinSize(t *testing.T) {
```

Comment:
I can do this for the simple case - empty accordion, but it gets tricky for non-empty accordion because the AccordionItem headers are buttons created by the renderer, so in order to measure them you need to access them via the renderer, hence internal tests.

Reply:
No, you don’t. You should not adjust your expectations to the implementation but the other way around. The text height is easily computable with a `canvas.Text`. Then the expected padding can be added via `theme.Padding`. There is no need to access the renderer nor to know the type of the actual element that is rendered.

Reply:
You're right. I added TestAccordionContainer_MinSize to accordion_test.go.

I think there is value in keeping this test as it is specific to accordionContainerRenderer implementation.